wasm/Makefile

```diff
@@ -0,0 +1,9 @@
+# Current build against rev 956f0d500c380cc1656a2d861b7ee12c2515a664 of https://github.com/proxy-wasm/proxy-wasm-cpp-sdk
+# Docker image: nojnhuh/proxy-wasm-cpp-sdk:956f0d500c380cc1656a2d861b7ee12c2515a664
```

Comment:
Should we be pushing this image to `openservicemesh` docker registry?

Reply:
Makes sense. I'll re-tag that image.